src/Nethermind/Nethermind.Consensus.AuRa/Validators/ReportingContractBasedValidator.PersistentReports.cs

```diff
@@ -91,7 +92,7 @@ private void ResendPersistedReports(BlockHeader blockHeader)
                         {
                             SendTransaction(ReportType.Malicious, _posdaoTxSender, CreateReportMaliciousTransactionCore(persistentReport));
                         }
-                        catch (AuRaException e)
+                        catch (AbiException e)
```

Comment:
in the other file you change AbiException to AuRa and here you changed the exception that you catch from AuRa to Abi. Was it intended?